lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -42,14 +42,12 @@ public function __construct($directory, $extension = self::EXTENSION)
      */
     protected function doFetch($id)
     {
-        $filename = $this->getFilename($id);
+        $value = @include $this->getFilename($id);
```

Comment:
*gasp*

Makes sense, but you might want to add a comment to these methods to the effect of your comment here. It's not immediately obvious from looking at the code.